operators/pkg/forge/resourcequota.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020-2021 Politecnico di Torino
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package forge
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	clv1alpha1 "github.com/netgroup-polito/CrownLabs/operators/api/v1alpha1"
+	clv1alpha2 "github.com/netgroup-polito/CrownLabs/operators/api/v1alpha2"
+)
+
+const (
+	// InstancesCountKey -> The key for accessing at the total number of instances in the corev1.ResourceList map.
+	InstancesCountKey = "count/instances.crownlabs.polito.it"
+)
+
+// TenantResourceList forges the Tenant Resource Quota as the sum of all quotas for each workspace.
+func TenantResourceList(workspaces []clv1alpha1.Workspace) clv1alpha2.TenantResourceQuota {
```

Comment:
This computation needs to take into account the overriding value too, as the status has to be correct in both cases,